### PR TITLE
Timesheet service db acces

### PR DESCRIPTION
**Pull request by shieran**
add service and entity for timesheet, access for database, CRUD methods

### CODE REVIEW

**Comment by shieran** [comment]
add service and entity for timesheet, access for database, CRUD methods #26

File: src/main/java/app/dao/impl/TimesheetDao.java [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    private static final String password = "root";
+    private static Connection connection;
+    private static PreparedStatement preparedStatement;
+    private static ResultSet resultSet;
+
+
+    public static String findById(int id) {
+        String query = "select * from timesheet where id = ?";
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
```
Comment (AlexKolonitsky) [inline src/main/java/app/dao/impl/TimesheetDao.java]: avoid copy past

**Review by AlexKolonitsky — CHANGES_REQUESTED** [review]

File: src/main/java/app/dao/impl/TimesheetDao.java (hunk 1 above)
Comment (AlexKolonitsky) [inline src/main/java/app/dao/impl/TimesheetDao.java]: move connection to static method and use it across all functions

**Review by AlexKolonitsky — CHANGES_REQUESTED** [review]

File: src/main/java/app/dao/impl/TimesheetDao.java (hunk 1 above)
Comment (shieran) [reply src/main/java/app/dao/impl/TimesheetDao.java]: done

Comment (shieran) [reply src/main/java/app/dao/impl/TimesheetDao.java]: connection is taken from class JDBCConection now

**Review by AlexKolonitsky — CHANGES_REQUESTED** [review]
use hibernate and common interface for dao

**Pull request closed without merging**